src/Microsoft.AspNet.Razor/Parser/TokenizerBackedParser.cs

```diff
@@ -75,6 +76,39 @@ protected void Initialize(SpanBuilder span)
             }
         }
 
+        protected TSymbol Lookahead(int count = 1)
+        {
+            if (count == 0)
+            {
+                throw new ArgumentOutOfRangeException(nameof(count));
```

Comment:
I'd rather keep Lookahead throwing when its 0, i'll change the condition to be `<= 0`